ui/src/organizations/containers/OrgTasksIndex.tsx

```diff
@@ -0,0 +1,140 @@
+import React, {Component} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+import {connect} from 'react-redux'
+
+// Components
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import OrganizationNavigation from 'src/organizations/components/OrganizationNavigation'
+import OrgHeader from 'src/organizations/containers/OrgHeader'
+import {Tabs} from 'src/clockface'
+import {Page} from 'src/pageLayout'
+import {SpinnerContainer, TechnoSpinner} from '@influxdata/clockface'
+import TabbedPageSection from 'src/shared/components/tabbed_page/TabbedPageSection'
+import OrgTasksPage from 'src/organizations/components/OrgTasksPage'
+
+//Actions
+import * as NotificationsActions from 'src/types/actions/notifications'
+import * as notifyActions from 'src/shared/actions/notifications'
+import {getTasks as getTasksAction} from 'src/organizations/actions/orgView'
+
+// Types
+import {Organization} from '@influxdata/influx'
+import {AppState, Task} from 'src/types/v2'
+import {RemoteDataState} from 'src/types'
+
+interface RouterProps {
+  params: {
+    orgID: string
+  }
+}
+
+interface DispatchProps {
+  notify: NotificationsActions.PublishNotificationActionCreator
+  getTasks: typeof getTasksAction
+}
+
+interface StateProps {
+  org: Organization
+  tasks: Task[]
+}
+
+type Props = WithRouterProps & RouterProps & DispatchProps & StateProps
+
+interface State {
+  loadingState: RemoteDataState
+}
+
+@ErrorHandling
+class OrgTasksIndex extends Component<Props, State> {
+  public state = {
+    loadingState: RemoteDataState.NotStarted,
+  }
+
+  public componentDidMount = async () => {
+    this.setState({loadingState: RemoteDataState.Loading})
```

Comment:
Could this loading state exist in Redux instead? Since the data is loaded by a Redux action, I think it makes sense that the loading state would be stored in Redux as well.

It also enables the pattern of connecting the loading state to the component and doing something like:

```
if (loadingState === RemoteDataState.NotStarted) {
  getTasks()
}
```

That way we only load tasks if they haven't already been loaded. This is useful since the tasks are used on multiple pages, and we don't need to reload them when switching pages.

Reply:
Also kinda related but not just specific to tasks... when we fetch data through a fetch-and-cache Redux action, we should probably have some way of limiting the concurrency of that action. I.e. if we call `getTasks` twice in a row, the requests will both be setting Redux state. I would expect instead that the first call gets canceled, and the state only gets set once. This will become more important if we store loading states in Redux as well.

Reply:
I will consider and implement this in the next PR.